Backend/tests/test_agent_integration.py

```
@@ -0,0 +1,104 @@
+import requests
+import json
+import time
+
+BASE_URL = "http://localhost:5000"
+
+def test_agent_context():
+    """Test the agent context endpoint."""
+    print("Testing agent context endpoint...")
+    
+    try:
+        response = requests.get(f"{BASE_URL}/agent/context?user_id=default")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)